src/main/java/taxi/util/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package taxi.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't found MySQL driver", e);
+        }
+    }
```

Comment:
You can safely delete this static initialization. But it is not necessary

Reply:
Thanks for advice, but in my case were problem with connections and mentor suggest back this static block